core/api/service/state/impl/state_api_impl.cpp

```diff
@@ -33,4 +36,9 @@ namespace kagome::api {
     OUTCOME_TRY(trie_reader, storage_->getEphemeralBatchAt(header.state_root));
     return trie_reader->get(key);
   }
+
+  outcome::result<primitives::Version> StateApiImpl::getRuntimeVersion(
+      std::optional<primitives::BlockHash> const &at) const {
```

Comment:
this is not critical, but I would like to keep code style consistent within a project. So can you please put `const` to the beginning of argument declaration?

Reply:
Ok. But IMO such format is more logical. Because const is a cv-qualifiers, it means that it is logical to locate it near the other qualifiers, for example ref-, to have the format <Type> [<Qualifiers>...] name;

Reply:
I understand. But this is how most of the people write it nowadays. And in the project we should be consistent and either change codestyle in all other places or change it here